notebooks/Shuvomoy - Benders decomposition.ipynb

```diff
@@ -390,7 +401,7 @@
     "if statusMasterProblem == :Optimal\n",
     "    fmCurrent = getValue(t)\n",
     "    xCurrent=Float64[]\n",
-    "    for i in 1:dimX\n",
+    "        for i in collect(1:dimX)\n",
```

Comment:
Seems that when I had 
`for i in [1:m]
do ...
end`
it produced the concatenation message.
When it is
`for i in 1:10
do ... 
end` 
or
`for i=1:10
do ...
end` 

it works fine. So, I will make the changes accordingly, and remove the collect()s.